packages/app-review/android/src/main/java/io/capawesome/capacitorjs/plugins/appreview/AppReviewPlugin.java

```diff
@@ -1,22 +1,67 @@
 package io.capawesome.capacitorjs.plugins.appreview;
 
 import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
 import com.getcapacitor.Plugin;
 import com.getcapacitor.PluginCall;
 import com.getcapacitor.PluginMethod;
 import com.getcapacitor.annotation.CapacitorPlugin;
+import com.google.android.play.core.review.ReviewInfo;
 
 @CapacitorPlugin(name = "AppReview")
 public class AppReviewPlugin extends Plugin {
 
-    private AppReview implementation = new AppReview();
+    private AppReview appReview;
+    private ReviewInfo reviewInfo;
+
+    @Override
+    public void load() {
+        appReview = new AppReview(getActivity());
```

Comment:
```suggestion
        appReview = new AppReview(this);
```

Please just pass the plugin instance, example (ignore `config`): https://github.com/capawesome-team/capacitor-plugins/blob/d4e76b7f64b165b703667bfb0e5e67e2a543f7e8/packages/live-update/android/src/main/java/io/capawesome/capacitorjs/plugins/liveupdate/LiveUpdatePlugin.java#L52